Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,14 @@
-import React from 'react';
-import './Card.css';
+import React from "react";
+import "./Card.scss";
+
+const CardBanner = () => {
+  return (
+    <img
+      className="card-banner"
+      src="https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png"
+      alt="React Banner"
```

Comment:
Thanks for still using `alt`. It can be easy to stop those good practices as we go along, but using `alt` is important.